core/src/main/scala/cats/syntax/alternative.scala

```diff
@@ -51,6 +56,25 @@ final class SeparateOps[F[_], G[_, _], A, B](private val fgab: F[G[A, B]]) exten
                G: Bifoldable[G]): (F[A], F[B]) = A.separate[G, A, B](fgab)
 }
 
+final class SeparateFoldableOps[F[_], G[_, _], A, B](private val fgab: F[G[A, B]]) extends AnyVal {
+
+  /**
+   * @see [[Alternative.separateFoldable]]
+   *
+   * Example:
+   * {{{
+   * scala> import cats.implicits._
+   * scala> val l: List[Either[String, Int]] = List(Right(1), Left("error"))
+   * scala> l.separateFoldable
+   * res0: (List[String], List[Int]) = (List(error),List(1))
+   * }}}
+   */
+  def separateFoldable(implicit
```

Comment:
This enrichment method could be added to `SeparateOps` since the target has the same type.